services/net/content/ContentManager.cs

```diff
@@ -97,11 +107,13 @@ public override async Task RunAsync()
                     var topics = _options.GetContentTopics(dataSources
                         .Where(ds => ds.IsEnabled &&
                             ds.ContentTypeId > 0 &&
-                            !String.IsNullOrWhiteSpace(ds.Topic) &&
-                            ds.Connection.ContainsKey("import") &&
-                            ((JsonElement)ds.Connection["import"]).GetBoolean())
+                            !String.IsNullOrWhiteSpace(ds.Topic))
                         .Select(ds => ds.Topic).ToArray());
 
+                    // Only include topics that exist.
+                    var kafkaTopics = this.KafkaAdmin.ListTopics();
```

Comment:
Now that we assume any data source with a `DataSource.ContentTypeId > 1` is importing, we need to remove all the non-existing topics.